packages/gestalt/src/Contents.css

```diff
@@ -23,6 +23,10 @@
   min-width: 180px;
 }
 
+.minDimensionsCustom {
+  padding: 2px
```

Comment:
@NEsfandiari we should not add this padding when `< 180` - it feels too much like magic.